assets/components/FormSection.tsx

```diff
@@ -1,8 +1,14 @@
 import React, {useState} from 'react';
-import PropTypes from 'prop-types';
 
-export function FormSection({name, testId, children}) {
-    const [opened, setOpened] = useState(name == null);
+interface IProps {
+    initiallyOpen?: boolean;
+    name?: string;
+    testId: string;
+    children: JSX.Element;
+}
+
+export function FormSection({initiallyOpen, name, testId, children}: IProps) {
+    const [opened, setOpened] = useState(initiallyOpen ?? name == null);
```

Comment:
maybe this should be actually `true` by default, seem to me like it's always the case now

Reply:
For now yes, all the usages are in places where it has to be open initially, I just thought I'd future proof a little bit by sticking with the common pattern.